src/styled.js

```diff
@@ -10,11 +10,11 @@ const setPragma = pragma => (h = pragma);
 function styled(tag) {
   const _ctx = this || {};
 
-  return function() {
+  return function () {
     const _args = arguments;
 
     return function Styled(props) {
-      _ctx.p = props || {};
+      _ctx.p = props ? Object.assign({}, props) : {};
```

Comment:
Should we just use the assign call directly? Maybe that'll shave some more bytes 🙂

Reply:
I believe that should be possible. The props argument is always an object for me, so checking it is not required.